src/spsc/mod.rs

```diff
@@ -83,42 +83,32 @@
 //! - The numbers reported correspond to the successful path (i.e. `Some` is returned by `dequeue`
 //! and `Ok` is returned by `enqueue`).
 
-use core::{cell::UnsafeCell, fmt, hash, marker::PhantomData, mem::MaybeUninit, ptr};
+use core::{cell::UnsafeCell, fmt, hash, mem::MaybeUninit, ptr};
 
-use generic_array::{ArrayLength, GenericArray};
 use hash32;
 
 use crate::sealed::spsc as sealed;
 pub use split::{Consumer, Producer};
 
 mod split;
 
-/// Multi core synchronization - a memory barrier is used for synchronization
-pub struct MultiCore;
-
-/// Single core synchronization - no memory barrier synchronization, just a compiler fence
-pub struct SingleCore;
-
-// Atomic{U8,U16, Usize} with no CAS operations that works on targets that have "no atomic support"
+// Atomic{U8, U16, Usize} with no CAS operations that works on targets that have "no atomic support"
```

Comment:
Unfortunately U8 and other consts should be fixed in docs.

Reply:
They should be fixed, this is part of the `Atomic`'s type signature which has `Uxx` in its name.